src/database/db_build.js

```diff
@@ -0,0 +1,8 @@
+const dbcon = require('./db_connect.js');
+const fs = require('fs');
+const sql = fs.readFileSync(`${__dirname}/select_users.js`).toString();
```

Comment:
Shouldn't we read the db_build.sql?

Reply:
you should read from db_bulid.sql to read sql statment that build the table inside the database

Reply:
noted! Thanks for your sharp notice!